tools/run_tests/build_artifact_python.sh

```diff
@@ -39,15 +39,6 @@ export PIP=${PIP:-pip}
 export AUDITWHEEL=${AUDITWHEEL:-auditwheel}
 
 
-if [ "$SKIP_PIP_INSTALL" == "" ]
```

Comment:
Our setup_requires should take care of these.  SKIP_PIP_INSTALL was disabled for both mac and linux before this PR (and is not used for windows)